Fixes go1.19 error on i686 - 
    runtime/cgo(.text): unknown symbol __stack_chk_fail_local in pcrel

--- a/go/src/cmd/go/script_test.go
+++ b/go/src/cmd/go/script_test.go
@@ -148,6 +148,7 @@ var extraEnvKeys = []string{
 	"GO_TESTING_GOTOOLS", // for gccgo testing
 	"GCCGO",              // for gccgo testing
 	"GCCGOTOOLDIR",       // for gccgo testing
+	"CGO_CFLAGS",         // for working around go1.19 "unknown symbol __stack_chk_fail_local" error on x86
 }
 
 // setup sets up the test execution temporary directory and environment.
